### PR TITLE
first pass at adding markup for content acces/entitlement levels

### PR DESCRIPTION
**Pull request by tavvy**
**Adds entitlementsLevel markup to content pages**
- required for our collaboration with google on the "subscribe with google project"
- this has been agreed by schema.org as the new markup for denoting content that has entitlement requirements
- essentially we have name spaced "products"

@GlynnPhillips tagging you in this as I haven't been in this code before and want to make sure I took the right approach.

screen shot 2018-03-02 at 11 05 59

 🐿 v2.5.16

### CODE REVIEW

**Comment by tavvy** [comment]
@ myself, need to add the `isPartOf` markup to barriers (in front of articles) too.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,92 @@
...
+> Markup your pages with a `productID` and publication name. This markup:
+>
+> * Globally identifies your publication to SwG
+> * Indicates the entitlement required to access the content
+> * Indicates whether or not this particular document requires entitlement
+> * Helps swg.js check access to the content
+
+> The markup is an extension of the existing Subscription and paywalled content markup, which you should be using to indicate restricted content to Google.  The markup must include publisher property.
+
+> **Note**: A `productID` can be changed, but the new `productID` should be added to SKUs before changing the markup on the page. After a grace period to ensure full indexing, the deprecated `productID` can be removed from relevant SKUs.
```
Comment (remybach) [inline README.md]: Is there any info on how long this grace period is?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,92 @@
...
+Firstly it is important to understand that each piece of content has an `accessLevel` associated with it. The access levels / restrictions for **content** are `free`,`registered`,`subscribed`,`premium`.
+
+These map nicely to Google's idea of **user** "entitlements". Whereby a **user** with an `ft.com:subscribed` entitlement would be allowed access to `subscribed` content and everything below (+ `registered` + `free`).
+
+#### Our entitlements range productIDs/labels:
+
+- **1 `ft.com:free`** (access to content with a `free` accessLevel)
+- **2 `ft.com:registered`** (`registered` accessLevel content + `free`)
+- **3 `ft.com:subscribed`** (`subscribed` accessLevel content + `registered` + `free`)
+- **4 `ft.com:premium`** (`premium` accessLevel content + `subscribed` + `registered` + `free`)
```
Comment (remybach) [inline README.md]: *Total* nitpick, but could the numbers be moved outside the bolding to make a proper `<ol>`? i.e.: `1. **...`

File: README.md [hunk 3]
```diff
@@ -1,3 +1,92 @@
...
+	'isAccessibleForFree': 'True',
+	'isPartOf': {
+		'@type': [ 'CreativeWork', 'Product' ],
+		'name' : 'Financial Times',
+		'productID': 'ft.com:free'
+	}
+}
+```
+
+#### Content Barrier/Paywall
```
Comment (remybach) [inline README.md]: Ah, so are we still doing this on barriers when there's a content uuid passed through? I thought we'd always be manually initialising regardless?

Comment (tavvy) [reply README.md]: yep, the main reason being it unlocks the `getEntitlements` check that we will need to do on content paywalls

File: README.md (hunk 1 above)
Comment (tavvy) [reply README.md]: copied directly from googles docs im afraid

**Pull request merged**